### DISCUSSION TITLE
Can't create asyncapi docs for nested Pydantic models

### DISCUSSION THREAD

**Discussion by dumpler** [ag2ai/faststream#867] [Q&A] [answered]
Hello, the problem is that when using nested Pydantic models, I get the error "Error: Token "$defs"does not exist." when generating AsyncAPI documentation. What am I doing wrong or is it a bug?

**Source code**
```import uvicorn
from fastapi import FastAPI

from faststream import Logger
from faststream.kafka import KafkaBroker
from faststream.kafka.fastapi import KafkaRouter
from pydantic import BaseModel

router = KafkaRouter()
router.broker = KafkaBroker("localhost:9094")
logger = Logger("test")
app = FastAPI(lifespan=router.lifespan_context)


class Foo(BaseModel):
    bar: str


class Baz(BaseModel):
    foo: Foo


@router.subscriber("in")
@router.publisher("out")
async def handle(baz: Baz):
    logger.info(baz)
    return "Response"


@router.subscriber("out")
async def handle_response(baz: Baz):
    logger.info(baz)


if __name__ == "__main__":
    uvicorn.run(app, port=8000, host="0.0.0.0", log_level="info")
```

**Expected behavior**
Correct AsyncAPI page, but I'm not sure if my code is correct

**Observed behavior**
Get "Error: Token "$defs"does not exist." at 0.0.0.0:8000/asyncapi

**Environment**
python = "3.11"
fastapi = "0.104.0"
faststream = {extras = ["kafka"], version = "0.2.5"}
pydantic = "2.4.2"
uvicorn = "0.23.2"

**Comment by Lancetnik** [comment, marked as answer]
@dumpler thanks for the report! Looks like this is a bug indeed. I created an Issue for it, will close it soon: https://github.com/airtai/faststream/issues/868

**Reply by dumpler** [reply]
Thanks for the quick response

**Reply by Lancetnik** [reply]
Looks like the job is done, the team will merge it tomorrow, I think

**Reply by dumpler** [reply]
It's so fast, thank you

**Reply by davorrunje** [reply]
The new release with the bug fix is out: https://github.com/airtai/faststream/releases/tag/0.2.6
